VERSION

```diff
@@ -1 +1 @@
-1.2.1
+vUDK2017
```

Comment:
I assume we are starting with the very first tag of EDK2 and will gradually add support for newer versions? vUDK2017 is so old...

Reply:
@miczyg1 no, this would be economically not feasible. We adding what is needed, not necessarily in chronological order: https://github.com/3mdeb/edk2-docker/pull/9/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R9